drivers/i2s/Kconfig.esp32

```diff
@@ -31,4 +31,10 @@ config I2S_ESP32_DMA_DESC_NUM_MAX
 	help
 	  Max number of link descriptor available for DMA transfers on each I2S channel
 
+config I2S_ESP32_ALLOWED_EMPTY_TX_QUEUE_DEFERRAL_TIME_MS
+	int "ESP32 I2S number of link descriptors"
```

Comment:
`int "ESP32 I2S number of link descriptors"`
The description does not fit the config itself, which is time related.

Reply:
is 0 a good default value? It means `k_timer_start` will start it without any waiting interval.